web-wallet/src/routes/(app)/settings/__tests__/page.spec.js

```diff
@@ -75,6 +88,21 @@ describe("Settings", () => {
 		expect(resetButton).toHaveAttribute("data-tooltip-disabled", "false");
 	});
 
+	it("should disable the \"Back\" button if invalid gas limit or price are introduced", async () => {
+		const { getByLabelText, getByRole } = render(Settings, {});
+		const priceInput = asInput(getByLabelText(/price/i));
+		const limitInput = asInput(getByLabelText(/limit/i));
+		const backButton = getByRole("link");
+
+		await fireInput(priceInput, 30000000);
+		expect(backButton).toHaveAttribute("aria-disabled", "true");
+		await fireInput(priceInput, 20000000);
+
```

Comment:
Aren't you missing an expectation here?

Reply:
I was resetting the input value so that the snapshot is not modified.

Reply:
But now that you mention it i can add a check to see if the button is enabled again

Reply:
> I was resetting the input value so that the snapshot is not modified.

Not getting the bit about the snapshot (and I don't see any here), but my point was that this is the right place to add an expectation where you check that the button isn't disabled anymore.

One thing I missed before: I would pair the "aria-disabled" attribute check with the `toBeEnabled` / `toBeDisabled` matchers.

Reply:
> One thing I missed before: I would pair the "aria-disabled" attribute check with the `toBeEnabled` / `toBeDisabled` matchers.

This cannot be done as those functions do not exist on type `HTMLElement`

Reply:
Ah, that's an anchor (`AnchorButton`), right.